Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -0,0 +1,158 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import Shared
+import BraveShared
+
+enum PlaylistPopoverState {
+    case addToPlaylist
+    case addedToPlaylist
+}
+
+struct PlaylistPopoverView: View {
+    @State
+    var state: PlaylistPopoverState
+    
+    var onPrimaryButtonPressed: (() -> Void)?
+    var onSecondaryButtonPressed: (() -> Void)?
```

Comment:
Could these buttons be better named? primary and secondary does not tell me much when I look at it

Reply:
Well this one popover is used for 2 different UI atm. In one case, it is used for onboarding and the buttons are different. In the other case, it is used in URL bar and the buttons are different :(